bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Cursor.java

```diff
@@ -59,9 +62,18 @@ public final class Cursor extends Resource {
 	 * @noreference This field is not intended to be referenced by clients.
 	 */
 	public long handle;
```

Comment:
Why not make this one `private` and let every consumer go through the new static method `win32_getHandle(...)`?